router/xgress_edge_tunnel/dialer.go

```diff
@@ -51,7 +52,7 @@ func (self *tunneler) Dial(destination string, circuitId *identity.TokenId, addr
 
```

Comment:
Above, where `self.terminators.get(destination)` returns an error if the terminator is not found is where we'd want to return an InvalidTerminator error